src/inversion-of-control/container-manager.ts

```diff
@@ -13,4 +14,6 @@ polarisContainer.bind(POLARIS_TYPES.GraphQLServer).to(PolarisGraphQLServer);
 polarisContainer.bind(POLARIS_TYPES.GraphQLLogger).to(PolarisGraphQLLogger);
 polarisContainer.bind(POLARIS_TYPES.Middleware).to(PolarisMiddleware);
 polarisContainer.bind(POLARIS_TYPES.Middleware).to(IrrelevantEntitiesMiddleware);
+polarisContainer.bind(POLARIS_TYPES.RealitiesHolderValidator).to(RealitiesHolderValidator);
+polarisContainer.bind(POLARIS_TYPES.RealitiesHolder).toConstantValue({});
```

Comment:
supply with normal object (that implements realityiesholder but return always true)